### ISSUE TITLE
workspace/load-session command does not exist

### ISSUE THREAD

**Issue by zedth2** [hlissner/doom-emacs#6201]
### What did you expect to happen?

I was trying to find the documentation for how to handle workspaces. My workspaces I think are getting saved in project folders? So I was attempting to figure out how to load workspaces from files specifically. Which lead me into the  [`modules/ui/workspaces/README.org`](https://github.com/hlissner/doom-emacs/blob/master/modules/ui/workspaces/README.org) but they list a command of `+workspace/load-session` which does not exist.

### What actually happened?

I remained confused by the documentation?

### Describe your attempts to resolve the issue

I think the correct answer is doom/load-session? But I'm not really sure.

### Steps to reproduce

1. Open Doom Emacs
2. M-x load-session
3. See that the only load-session is doom/load-session with a TODO on it?

### System Information

https://pastebin.com/nu0w8EVT

**Comment by elken** [comment]
A simple typo most likely, `+workspace/load` is what you want as per the table below.

**Comment by zedth2** [comment]
> A simple typo most likely, `+workspace/load` is what you want as per the table below.

Yeah that just gives me some list and I can't figure out where it's pulling that list from. It doesn't let me chose a specific file.

**Comment by elken** [comment]
Evaluate `(expand-file-name +workspaces-data-file persp-save-dir)` to give you the file it's pulling from

**Comment by zedth2** [comment]
Yeah found that file don't know where all my workspace/save command went to. I have an autosave in my project home for persp but no idea how to force it to load that one. Guess I'll go read projectiles documentation next.

**Comment by elken** [comment]
`SPC h f` will let you introspect functions and look at their source code, it's one of the most powerful capabilities of Emacs :)

It seems like there is no issue here however (other than the typo, of which I will mention to Henrik as the docs are currently being refactored) so if you're happy then please close this :)

**Comment by zedth2** [comment]
After digging a bit more I'm not sure the author meant workspace/load. There seems to be a fundamental difference between sessions and workspaces. I haven't been able to figure it out. Also why would one close an issue before actually fixing the source in the repo to resolve the issue?
